core/src/components/chip/test/shape/chip.e2e.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  /**
+   * This behavior only applies to Ionic Theme.
+   */
+  test.describe(title('chip: shape'), () => {
+    test.describe('default', () => {
+      test('should not have visual regressions', async ({ page }) => {
+        await page.goto(`/src/components/chip/test/shape`, config);
```

Comment:
Optional: Since we are going to the same page for every test, we can set this goto we can set this in a [beforeEach](https://playwright.dev/docs/api/class-test#test-before-each) inside of the `chip: shape` describe block.

Reply:
Thanks, didn't knew this, already changed to use it!